dialogue-apache-hc4-client/src/test/java/com/palantir/dialogue/hc4/WeakSummingGaugeTest.java

```diff
@@ -24,7 +24,7 @@
     @Test
     public void empty_initially() {
         WeakSummingGauge<String> gauge = new WeakSummingGauge<>(String::length);
-        assertThat(gauge.getValue()).isEqualTo(0);
+        assertThat(gauge.getValue()).isZero();
```

Comment:
Thoughts on undoing this refactoring? I don't think it really improves the error message at all and just results in some extra emails...

Reply:
Happy to disable it